src/Particular.PlatformSample/Particular.PlatformSample.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```

Comment:
Given we're making a change anyway, and that netcoreapp2.1 is out of support, should netstandard be changed to netcoreapp3.1 ?